numpy/linalg/Transformations2D.py

```diff
@@ -0,0 +1,41 @@
+#2D matrices used for linear transformation in linear algebra 
+
+import numpy as np
+import math 
+
+class Transformations:
```

Comment:
Some general feedback for if you submit this elsewhere: this class has no `__init__`, and none of the methods use their `self` argument &mdash; this is a strong indication that you should not be using a class at all.

Reply:
Thank you! I will definitely do the needful.